### ISSUE TITLE
Option to change order of exported data

### ISSUE THREAD

**Issue by JaneConvergine** [verbb/formie#2107]
### What are you trying to do?

We would like to see an option to reorder the exported data for form submissions in the Formie plugin. Currently, when form data is exported to a CSV file, it appears in a "newest to oldest" order, with the newest submissions at the top. Many organizations may prefer to review exported data with the oldest entries at the top (sorted oldest to newest) to align with their data review processes.

### What's your proposed solution?

A few possible solutions that would address this request:
**1. Default Sort Order Option:** Add an option in the Formie plugin to set the default sort order for CSV exports to either "oldest to newest" or "newest to oldest."
**2. Follow In-App Sort Order:** Enable the exported data to follow the sort order set within Craft CMS when viewing form submissions. For instance, if we have sorted or filtered data in a specific way within Craft (e.g., by date or by checkbox selection), the CSV export should respect that sorting/filtering.

These options would provide greater flexibility for various organizations to customize their data exports according to specific needs and workflows.

### Additional context

The ability to control the sort order in the export file would allow organizations to streamline data review processes, reducing the need for additional manual reordering. This feature could support teams in maintaining consistency in data presentation and better adapting Formie exports to fit existing workflows.

Thank you for considering this enhancement.

**Comment by engram-design** [comment]
I agree, and would love to add support for extra options for exporting. However, as we use Craft's element exporting functionality, we're unfortunately restricted to the UI they provide. We would have to essentially write our own handling of this to include this sort of thing, which is a bit of work, but likely the only solution going forward.

**Comment by engram-design** [comment]
Added in [4.0.0-beta.7](https://github.com/verbb/formie/releases/tag/4.0.0-beta.7), thanks to a revamped Reporting feature for exports.
